gef-ui/server/controllers/activities-controller/index.test.js

```diff
@@ -110,7 +101,20 @@ describe('mapPortalActivities', () => {
       },
     ];
 
-    expect(response).toEqual(expected);
+    expect(result).toEqual(expected);
+  });
+
+  describe('when author.lastName does not exist', () => {
+    it('should return a mapped array for mojTimeline without lastName in byline.text`', () => {
+      const mockActivity = dealSubmissionActivity;
+      mockActivity[0].author.lastName = '';
```

Comment:
This is reassigning the constant, you could get around this by either introducing a function `getDealSubmissionActivity` to create copies of this object or using clone deep here (https://www.geeksforgeeks.org/lodash-_-clonedeep-method/)

Reply:
I've just changed the way `mockActivity` is created, simpler IMO